app/javascript/src/components/Dashboard/Notes/constants.js

```diff
@@ -30,3 +25,76 @@ export const USER_AVATAR_URL = "https://picsum.photos/id/1005/5760/3840";
 export const SIXTY = 60;
 export const EIGHTY = 80;
 export const TWO_HUNDRED = 200;
+
+export const CONTACT_DETAILS_DATA = [
+  {
+    id: 0,
+    name: "Ronald Richards",
+  },
+  {
+    id: 1,
+    name: "Jacob Jones",
+  },
+  {
+    id: 2,
+    name: "Ronald Jones",
+  },
+  {
+    id: 3,
+    name: "Jacob Richards",
+  },
+];
+
+export const TAGS_DATA = [
+  {
+    id: 0,
+    name: "Getting Started",
+  },
+  {
+    id: 1,
+    name: "Onboarding",
+  },
+  {
+    id: 2,
+    name: "User Flow",
+  },
+  {
+    id: 3,
+    name: "UX",
+  },
+  {
+    id: 4,
+    name: "Bugs",
+  },
+  {
+    id: 5,
+    name: "V2",
+  },
+];
```

Comment:
https://bigbinary.slack.com/archives/C01T5Q4DZAR/p1663335627615309?thread_ts=1663060378.971029&cid=C01T5Q4DZAR

Reply:
The above changes were made. Thank you.